docs/devGuide/contributing.md

```diff
@@ -0,0 +1,72 @@
+   - Updating documentation
+   - Fixing bugs
+   - Implementing features 
+
+### How to report a bug
+
+Report bugs in our [issue tracker](https://github.com/MarkBind/markbind/issues), following the guideslines below. Following the guidelines helps us in undestanding your report better and reproduce the behavior more easily.
+
+- Has your bug been raised up before? Before reporting a bug, please search if it has been already reported in the existing issues.
+- Is your bug from the current version or an older version of MarkBind? What OS are you on? Tell us about your environment, for example, the MarkBind version you are on.
+- What did you do? What did you expect to happen? What actually happened? 
+  - Include the actual MarkBind code causing the issue, the steps for reproduction, the actual, raw output, and what you expect instead. This helps us to reproduce the bug to understand it better. 
+  - You are also encouraged to submit a PR that reproduces this in `test\functional\test_site\bugs\`.
+
+Please do not submit personal support requests as bugs, [submit a blank issue](https://github.com/MarkBind/markbind/issues/new) to our issue tracker and label it with <a href="https://github.com/MarkBind/markbind/issues?q=is%3Aopen+is%3Aissue+label%3Aquestion" class="badge" style="color:white; background-color: #A41BD6;">question</a>.
+
+### How to request a feature
+
+Request for new features or enhancements to existing features in our [issue tracker](https://github.com/MarkBind/markbind/issues), following the guideslines below. Following the guidelines helps us in undestanding what your problem or use case is, so that contributors may know exactly what feature to implement.
+
+- Has someone already requested your feature before? Before requesting a new feature, please search if it has been already requested in the existing issues.
+- Is your request related to a problem? Provide a clear and concise description of what the problem is. Eg: I have an issue when [...]
+- Describe the solution you'd like. Provide a clear and concise description of what you want to happen.
+- Describe alternatives you've considered. Let us know about other solutions you've tried or researched.
+- If any, share some additional context. Is there anything else you can add about the proposal? You might want to link to related issues here if you haven't already.
+
+### First issues for contributors
+
+Unsure where to begin contributing to MarkBind? You can start by looking through these issues marked <a href="https://github.com/MarkBind/markbind/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+sort%3Acomments-desc" class="badge" style="color:white; background-color: #7057FF;">good first issue</a>.
+
+It is recommended you go through the [Getting Started](https://markbind.org/userGuide/gettingStarted.html) section in the User Guide, and play around with Markbind with your own basic site in order to explore and understand the various features it provides.
```

Comment:
`We recommend ... to play around with Markbind first. Explore and understand ...`

Reply:
Could also recommend visiting `devGuide.md` first to learn how to setup the dev environment, run tests, etc